src/hearings/hearings.routes.ts

```diff
@@ -72,6 +64,15 @@ export const ROUTES: Routes = [
           title: 'HMCTS Manage cases | Request Hearing | Date Priority Hearing'
         }
       },
+      {
+        path: 'location-search',
+        resolve: { hearingPriorities: PriorityResolver },
```

Comment:
reason to introduce this?
resolve: { hearingPriorities: PriorityResolver }, is PriorityResolver  used in your component?